plasmapy/physics/tests/test_distribution.py

```diff
@@ -91,10 +93,16 @@ def velDist3D(vx, vy, vz):
                           lambda z: infApprox,
                           lambda z, y: -infApprox,
                           lambda z, y: infApprox,
-                          epsabs=1.49e-08*u.m/u.m, 
-                          epsrel=1.49e-08*u.m/u.m)
-    exceptStr = "Integral of distribution function should be 1."
-    assert np.isclose(integ.value, 1), exceptStr
+                          epsabs=0.0, 
+                          epsrel=5.0e-1)
+    # value returned from tplquad is (integral, error), we just need the 1st
```

Comment:
Or we could use the returned error in line 102-105, as `atol`? Does that make sense?

Reply:
I'm not sure. I don't think the error here is the error on the final value, but rather the error in how the function slices up the bounds into individual units to be calculated, or the error on these local approximate integrals.

Reply:
In any case using `epabs=5.0e-1` gave me quite a reasonable value on the final integral value when I stripped the Maxwellian function of units. This was the aforementioned 1.0000000181399045.